.circleci/config.yml

```diff
@@ -17,15 +17,6 @@ jobs:
       - android/run-tests:
           test-command: ./gradlew testDebug
       - android/save-gradle-cache
-  android_test:
-    executor:
-      name: android/android-machine
-      resource-class: large
-      tag: default
-    steps:
-      - checkout
-      - android/start-emulator-and-run-tests:
-          system-image: system-images;android-30;google_apis;x86
```

Comment:
I dropped the android tests because

(a) the smoke tests make them functionally obsolete  
(b) getting them to work would require _also_ spinning up a mock server, and that just made them look even _more_ like the smoke tests

Reply:
Makes sense to me